index.js

```diff
@@ -19,12 +20,13 @@ ctx.strokeStyle = "#000";
 
 //Load image
 const imageObj = new Image();
+imageObj.crossOrigin = "anonymous"
```

Comment:
Leave a comment why this is necessary?

Reply:
https://stackoverflow.com/questions/23123237/drawing-images-to-canvas-with-img-crossorigin-anonymous-doesnt-work